Makefile

```diff
@@ -33,6 +27,7 @@ SRC = $(wildcard src/*.c)
 OBJS = $(patsubst $(SRC_DIR)/%.c, $(BUILD_DIR)/%.o, $(SRC))
 
 PG_VERSION = $(strip $(shell $(PG_CONFIG) --version | $(GREP) -oP '(?<=PostgreSQL )[0-9]+'))
+# 0 is true
```

Comment:
This is confusing; these vars read like a bool but the values are inverted (0 is true, 1 is false). I'd consider flipping them.

Reply:
@soedirgo This is how shell works https://stackoverflow.com/questions/2933843/why-0-is-true-but-false-is-1-in-the-shell